src/pages/home_page/sub_pages/Gallery.tsx

```diff
@@ -113,23 +105,24 @@ const Gallery = (): JSX.Element => {
             }}
         >
             <ImageGrid
-                img_one={one}
-                img_two={two}
-                img_three={three}
-                img_four={four}
-                img_five={five}
-                img_six={six}
+                img_one={imgData?.img_one}
```

Comment:
Make the naming dynamic. This `img_one` etc works because you uploaded them with that name, but think of the case when an end user uploads images. Then naming will be different.